src/test/garfile/review/post.controller.test.js

```diff
@@ -243,7 +243,7 @@ describe('GAR Review Post Controller', () => {
 
       callController().then().then().then()
         .then(() => {
-          expect(res.render).to.have.been.calledWith('app/garfile/review', { cookie });
+          expect(res.render).to.have.been.calledWith('app/garfile/review/index.njk', { cookie });
         });
     });
 
```

Comment:
Nit: Delete commented tests --> refers to code which is not longer needed + version control can get it back:
```javascript
// expect(res.render).to.have.been.calledWith('app/garfile/submit/success/index', { cookie });
```